src/reps/grip.js

```diff
@@ -243,4 +243,6 @@ let Grip = {
 };
 
 // Exports from this module
-module.exports = Grip;
+module.exports = {
+  Grip: Grip
```

Comment:
Sure! I just wanted to go back to what was on MC, since I knew that should work :)